src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -10,7 +10,8 @@ $header-height: 45px;
   flex-flow: column;
   top: 66px;
   right: 0;
-  width: 360px;
+  width: $sidebar-width;
+  min-width: $sidebar-width;
   height: calc(100vh - 96px);
```

Comment:
What does this `96px` 'magic number` represent here, and do we have an SCSS variable for it yet? Is it the header height or similar?

Also, is it possible to change the `100vh` here to `100%` and still have it look the same?

Reply:
well, `navbar-margin` styles contains 
```
padding-top: 66px;
padding-bottom: 30px;
```
That's why here is using 96px. Changing to `100%` will cut the chat for small resolution

Reply:
I meant literally replacing the 100vh with 100%, yet still keeping the calc/etc logic. Does that work?

Do we have an scss variable for that 96px of navbar height set anywhere? Can we use that here?

Reply:
Looking at `src/components/organisms/WithNavigationBar/WithNavigationBar.scss`:

```scss
.navbar-margin {
  padding-top: $navbar-height;
  padding-bottom: $footer-height;
}
```

Looking at `src/scss/constants.scss`:

```scss
$navbar-height: 66px;
$footer-height: 30px;
```

So we have variables for the `NavBar` height, as I guessed.

---

Therefore, within `src/components/organisms/ChatSidebar/ChatSidebar.scss` I'm wondering if we can set the styles like this:

```diff
- top: 66px;
+ top: $navbar-height;
  right: 0;
  width: $sidebar-width;
  min-width: $sidebar-width;
- height: calc(100vh - 96px);
+ height: calc(100% - $navbar-height - $footer-height);
```

I just tested in DevTools and it still seems to look correct to me when using 100% (since when floating it's going to be 100% of I believe the `body` or `html` or similar tags, which are set in a way that they should be more or less equivalent to `100vh` (which means '100% of the viewport height'). But by using the `%` instead of `vh`, if we embed this within another container and/or when we make it 'non-floating', then it should end up working better. In the 'non-floating' mode we'll likely want/need to set the height to just `100%`, as we'll likely no longer need to account for the header/footer explicitly.

Reply:
Yes, it look better now, thanks

Reply:
Awesome :)